src/state/slices/portfolioSlice/selectors.ts

```diff
@@ -204,6 +204,22 @@ export const selectPortfolioTotalFiatBalance = createSelector(
       .toFixed(2),
 )
 
+export const selectPortfolioTotalFiatBalanceExcludeEarnDupes = createSelector(
+  selectPortfolioFiatBalances,
+  (portfolioFiatBalances): string => {
+    // ETH/FOX LP token and FOXy are portfolio assets, but they're also part of DeFi opportunities
+    // With the current architecture (having them both as portfolio assets and earn opportunities), we have to remove these two some place or another
+    // This obviously won't scale as we support more LP tokens, but for now, this at least gives this deduction a sane home we can grep with `dupes` or `duplicates`
```

Comment:
This is an intricately related product and engineering question - unless we get a consensus on how to handle this moving forward, we'll just have to keep on deducting dupes